packages/bot-web-ui/src/pages/dashboard/announcements/config.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
-import { localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
 import { DBOT_TABS } from 'Constants/bot-contents';
 import { handleOnConfirmAccumulator } from './utils/accumulator-helper-functions';
 import { IconAnnounce } from './announcement-components';
+import { OpenLiveChatLink } from '@deriv/components';
 
 export type TContentItem = {
     id: number;
-    text: string;
+    text: string | React.ReactElement;
 };
 
 export type TAnnounce = {
-    id: number;
+    id: string;
     main_title: string;
-    confirm_button_text: string;
-    cancel_button_text: string;
+    confirm_button_text?: string;
+    cancel_button_text?: string;
     base_classname: string;
-    title: string;
-    content: TContentItem[];
+    title: string | React.ReactElement;
```

Comment:
Same here, can we just use `React.ReactNode`?

Reply:
same here https://github.com/binary-com/deriv-app/pull/16734#discussion_r1749811738

Reply:
✅ done